opentelemetry-sdk/src/metrics/meter.rs

```diff
@@ -328,14 +340,20 @@ impl SdkMeter {
     {
         let validation_result = validate_instrument_config(builder.name.as_ref(), &builder.unit);
         if let Err(err) = validation_result {
-            global::handle_error(err);
+            otel_error!(
+                name: "InstrumentCreationFailed",
+                meter_name = self.scope.name.as_ref(),
+                instrument_name = builder.name.as_ref(),
+                message = "Measurements from this gauge will be ignored.",
```

Comment:
Let's use the capital letter for the first letter of the instrument name.

```suggestion
                message = "Measurements from this Gauge will be ignored.",
```